[OSDOCS#17012]: CQA bare metal ipi install cluster 3 (#117634)
openshift/openshift-docs: modules/ipi-install-following-the-progress-of-the-installation.adoc

[id="ipi-install-following-the-progress-of-the-installation_{context}"]
= Following the progress of the installation

[role="_abstract"]

> During the deployment process, you can check the installation's overall status by issuing the `tail` command to the `.openshift_install.log` log file in the install directory folder:

Reviewer: Still has a colon at the end of the sentence here:

```suggestion
During the deployment process, you can check the installation's overall status by issuing the `tail` command to the `.openshift_install.log` log file in the install directory folder.
```

.Procedure

> * Run the following command to check the installation overall status:

Reviewer: ```suggestion
* Track installation progress by running the following command:
```
